dev/simran0117/Sample.ipynb

```diff
@@ -0,0 +1,613 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Implementation of SVM and KNN on Wine Quality Dataset\n",
+    "\n",
+    "__importing python modules__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from sklearn.metrics import classification_report, confusion_matrix,accuracy_score, f1_score\n",
+    "from sklearn.svm import SVC  \n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.neighbors import KNeighborsClassifier \n",
+    "import mymodule\n",
+    "import Visualization\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "dataset=pd.read_csv(\"C://Users//Simran Verma//Outreachy//PRESC//datasets//winequality.csv\")"
```

Comment:
Please load the dataset from the correct relative path in the repo so that it runs without modification.

Reply:
Apologies, you had to point it out again. I have added the relative path.